cmd/state-custom-installer/docker/centos/runCompile.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+cd cli
+git pull
+# git checkout BE-1031-there-is-a-standalone-tool-for-unpacking-and-relocating-artifacts-in-installers 
+go mod vendor
```

Comment:
This should not happen in a script. The vendored files should be committed to the repo, not pulled in at build time.